src/bun.js/test/jest.zig

```diff
@@ -1737,3 +1772,187 @@ pub fn printGithubAnnotation(exception: *JSC.ZigException) void {
     Output.printError("\n", .{});
     Output.flush();
 }
+
+pub const EachData = struct { strong: JSC.Strong, is_test: bool };
+
+fn eachBind(
```

Comment:
This should really be two different functions or a single function with a comptime argument and then `is_test` becomes unnecessary

but that shouldn't block this from shipping